monai/networks/layers/factories.py

```diff
@@ -242,6 +245,25 @@ def sync_batch_factory(_dim) -> Type[nn.SyncBatchNorm]:
     return nn.SyncBatchNorm
 
 
+@Norm.factory_function("instance_nvfuser")
+def instance_nvfuser_factory(dim):
+    """
+    `InstanceNorm3dNVFuser` is a faster verison of InstanceNorm layer and implemented in `apex`.
+    It only supports 3d tensors as the input. It also requires to use with CUDA and non-Windows OS.
+    In this function, if the required library `apex.normalization.InstanceNorm3dNVFuser` does not exist,
+    `nn.InstanceNorm3d` will be returned instead.
+
+    """
+    types = (nn.InstanceNorm1d, nn.InstanceNorm2d, nn.InstanceNorm3d)
+    if dim != 3:
```

Comment:
I think this logic is not correct, if I pass `dim=2`, why use `dim-1` instead?

Thanks.

Reply:
Hi Nic, if pass `dim=2`, will use types[2-1] which corresponds to `nn.InstanceNorm2d`.